book/04-git-server/sections/gitlab.asc

```diff
@@ -38,7 +38,7 @@ image::images/gitlab-menu.png[O item Área de administração no menu GitLab.]
 Usuários no GitLab são contas que correspondem a pessoas.
 As contas de usuário não têm muita complexidade; A conta de usuário é uma coleção de informações pessoais anexadas aos dados de login.
 Cada conta de usuário vem com um *namespace*, que é um agrupamento lógico de projetos que pertencem a esse usuário.
-Se o usuário +jane+ tivesse um projeto chamado +project+, o URL do projeto seria http://servidor/jane/project[].
+Se o usuário +jane+ tivesse um projeto chamado +project+, o URL do projeto seria `http://servidor/jane/project`[].
```

Comment:
You can remove the trailing square brackets. They are not needed.

Reply:
I will address this in a future PR. Thank you